samples/web-workers/js/worker.js

```diff
@@ -6,26 +6,11 @@ import "./polyfill.worker";
 import { ContainerClient } from "@azure/storage-blob";
 
 async function uploadToStorageBlob() {
-  const azureStorageBlobConnectionString = process.env.AZURE_STORAGE_BLOB_CONNECTION_STRING;
-  if (!azureStorageBlobConnectionString) {
-    throw new Error(
-      "Required environment variable AZURE_STORAGE_BLOB_CONNECTION_STRING is either missing or empty."
-    );
-  }
-
-  const azureStorageBlobContainerName = process.env.AZURE_STORAGE_BLOB_CONTAINER_NAME;
-  if (!azureStorageBlobContainerName) {
-    throw new Error(
-      "Required environment variable AZURE_STORAGE_BLOB_CONTAINER_NAME is either missing or empty."
-    );
-  }
+  const containerSasUrl = "<your container SAS url>";
```

Comment:
Maybe worth putting a dummy url in a comment above the line?

Reply:
good point! added one